torchaudio/datasets/fluentcommands.py

```diff
@@ -0,0 +1,56 @@
+import csv
+import os
+from pathlib import Path
+from typing import Union
+
+import torchaudio
+from torch.utils.data import Dataset
+
+_FOLDER_IN_ARCHIVE = "fluent_speech_commands_dataset"
```

Comment:
Why should this be global variable? It is only used once, and it is not expected to be changed.